File: Progamming Languages/C++ VS/Person/Person/Person.cpp
#include<iostream>
#pragma once
using namespace std;
class Person {
private:
    string name;
    int age;
    float height;
    float weight;
    string address;
    
public:
    Person() {
        cout << "XIn chao" << endl;
    }
    Person(string name, int age,float height, float weight, string address)
    {
        this->name = name;
        this->age = age;
        this->height = height;
        this->weight = weight;
        this->address = address;
    }
    string getName() {
        return name;
    }
    int getAge() {
        return age;
    }
    string getAddress() {
        return address;
    }
    float getHeight() {
        return height;
    }
    float getWeight() {
        return weight;
    }
    virtual void display() {
        cout << "Name: " << getName() << endl;
        cout << "Age: " << getAge() << endl;
        cout << "Height: " << getHeight() << " m" << endl;
        cout << "Weight: " << getWeight() << " Kg " << endl;
        cout << "Add: " << getAddress() << endl;
    }
};

File: Progamming Languages/C#/OOP/exercise_8/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace exercise_8
{
    public class Person
    {
        private int id, age;
        private string name;

        public int Id{
            set{
                id = value;
            }

            get{
                return id;
            }
        }

        public int Age{
            get; set;
        }
        public string Name{
            get; set;
        }

        public string Address{
            get; set;
        }
        public void display(){
            Console.WriteLine("Id: " + Id);
            Console.WriteLine("Name: " + Name);
            Console.WriteLine("Age: " + Age);
            Console.WriteLine("Address: " + Address);
        }
    }
}